app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::PostsController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @post = current_user.posts
+    render json: @post, pretty: true
```

Comment:
- [ ] [OPTIONAL] As backend developers who have developed these API endpoints. I will like to commend you both for great job done, however I will advise that you be consistent with your JON response. The frontend developer will need a consistent response. Here is what I mean, on the comments controller, your JSON responses have this format:
https://github.com/zbahati/blogApp/blob/b930a4872f307e4d10e01aaaa6307fc7776845e9/app/controllers/api/v1/comments_controller.rb#L18
Which is a standard practice 👍🏽 , so your JSON response for the index action for the Lit of post too should be:

``` ruby
render json: { success: true, data: @post }, status: :ok
```
 Please, read more about this 👍🏽